frontend/components/impact-chart/component.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC } from 'react';
+
+import { Radar } from 'react-chartjs-2';
+
+import {
+  Chart as ChartJS,
+  RadialLinearScale,
+  PointElement,
+  LineElement,
+  Filler,
+  Tooltip,
+  ChartData,
+  ChartOptions,
+} from 'chart.js';
+
+import { ImpactChartProps } from './types';
+
+ChartJS.register(RadialLinearScale, PointElement, LineElement, Filler, Tooltip);
+
+export const ImpactChart: FC<ImpactChartProps> = ({ color, impact, labels }) => {
+  const data: ChartData<'radar'> = {
+    labels,
+    datasets: [
+      {
+        data: impact,
+        backgroundColor: color + 'cc',
```

Comment:
The background colour is not showing for me:
![Chart with no background colour](https://user-images.githubusercontent.com/6073968/168292669-68a9a4e4-cf90-4b41-92e9-4689a3f3eb71.png)

Reply:
fixed